web/pages/[[...slug]].tsx

```diff
@@ -77,10 +91,12 @@ const Route = ({
       page: { name, sections },
     },
     home: { name: homeName, startDate, endDate, description },
+    footer,
   },
   slug,
 }: RouteProps) => (
   <>
+    <main>
     {slug === '/' ? (
```

Comment:
Can these be indented? I'm not able to create a suggestion because some of the lines are "outside" the diff.

Reply:
(By "these" I meant the lines in between the `main` start/end tags.)

Reply:
Whoops, I forgot to run `prettier`. Annoying not to have a pre-push hook.. 6e925f7